crates/bevy_render/src/camera/projection.rs

```diff
@@ -112,3 +115,25 @@ impl Default for OrthographicProjection {
         }
     }
 }
+
+/// Given coordinates in world space, use the camera and window information to compute the
+/// screen space coordinates.
+pub fn world_to_screen(
+    world_space_coords: &GlobalTransform,
```

Comment:
I don't think we should limit ourselves to GlobalTransforms. A "point" could come from anywhere (ex: an arbitrary user calculation, something stored in an entity / resource, etc)

Reply:
I did waffle on whether I should constrain the input world space coordinates, I think you're right - I'll change this to a Vec3.